RELEASE_NOTES.md

```diff
@@ -1,5 +1,10 @@
 # Release Notes
 
+## Changes in Sawtooth Rust SDK 0.4.3
+
+* Stabilize the "transact-compat" feature by moving the feature flag from the
+  "experimental" feature group to the "stable" feature group in the Cargo.toml
```

Comment:
Nitpick:  Add "file" after "the Cargo.toml".  But it's a minor thing (not a blocker), so I'll approve this PR. If no other changes are needed, feel free to merge as is.